### PR TITLE
Feat/task

### PR DESCRIPTION
**Pull request by Gora5567**

### CODE REVIEW

File: helper.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def start_task(a):
+    print(f"\ntask{a}---> start\n")
+
+
+def end_task(a):
+    print(f"\n\tend <---task{a}\n")
```
Comment (megama5) [inline helper.py]: add an empty line to the end

File: string-processing-hw.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "Have you any wool?\n\n"
+      "Yes sir, yes sir,\n\n"
+      "Three bags full\n\n\n"
+      "One for the master,\n\n"
+      "One for the dame,\n\n"
+      "And one for the little boy\n\n"
+      "Who lives down the lane\n\n\n"
+      "Baa, baa, black sheep,\n\n"
+      "Have you any wool?\n\n"
+      "Yes sir, yes sir,\n\n"
```
Comment (megama5) [inline string-processing-hw.py]: Create a constant for "\n\n" string part

File: string-processing-hw.py
Comment (megama5) [inline string-processing-hw.py:1]: Create a separate function for each task

**Review by megama5 — APPROVED** [review]

**Comment by Gora5567** [comment]
nice

**Pull request merged**